src/app/personside/infotabs/saksoversikt/SakstemaVisning.tsx

```diff
@@ -131,7 +131,9 @@ class SakstemaVisning extends React.Component<SakstemaProps, State> {
     }
 
     render () {
-        const sakstema = this.props.sakstema;
+        const sakstema = this.props.sakstema.sort((a, b) => {
```

Comment:
`const sorterPåSisteHendelse = (a, b) => hentDatoForSisteHendelse(b).getTime() - hentDatoForSisteHendelse(a).getTime();
const sakstema = this.props.sakstema.sort(sorterPåSisteHendelse);` ..?